dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -2010,7 +2010,13 @@ private boolean internalDestroy (final List<Contentlet> contentlets, final User
                 throw new DotContentletStateException(CAN_T_CHANGE_STATE_OF_CHECKED_OUT_CONTENT);
             }
 
-            this.canLock(contentlet, user);
+            final boolean bringOldVersions  = false;  // we do not want old version in order to be more efficient
```

Comment:
I think this boolean declaration is not needed. The comment is enough

Reply:
I just want to be clearer on my intention, it is quite the same in terms of memory and so